interface/patient_file/encounter/delete_form.php

```diff
@@ -77,7 +77,7 @@
 ?>
 <input type="hidden" id="confirm" name="confirm" value="1"/>
 <p>
-<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . text($_GET['formname']); ?>
+<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . xlt(text($_GET['formname'])); ?>
```

Comment:
Note this is a special case since it is a form name. I do not expect you to know this stuff, and will flag it when I see them on the code reviews (these rarely come up). We have several special translations functions in case users wish to turn off translations for certain things(by default, they are all turned on):
xl_list_label
xl_layout_label
xl_gacl_group
xl_form_title
xl_document_category
xl_appt_category

And in this case since it's a form name, used xl_form_title:
https://github.com/openemr/openemr/commit/65efdba962415115e5fbea1663ca33022646accd

Again, you don't need to worry about these special functions; I'll bring them up on the reviews in the rare situation when it comes up.